Sources/Nakama/GrpcClient.swift

```diff
@@ -145,7 +146,9 @@ public class GrpcClient : Client {
         if vars != nil {
             req.account.vars = vars!
         }
-        return try await self.nakamaGrpcClient.authenticateCustom(req).response.get().toSession()
+        return try await retryInvoker.invokeWithRetry(request: {
```

Comment:
are we only using the retry invoker for authenticate custom at the moment?

Reply:
This is just for the use in the tests, after we merge there will be another PR to add retry code in all other grpc methods.

Reply:
ok sounds good 👍